30-recursion.cs

```diff
@@ -10,6 +10,7 @@
 static int GetFactorial(int number)
 {
     if (number >= 1)
```

Comment:
we know that 1! == 1 right, then why one more time uses recursion for n==1, and for your attention in recursive functions always exiting from recursion part goes in the first place. for example
```
if (number <= 1) {   return 1;  }  
return number*GetFactorial(number - 1);
```